### DISCUSSION TITLE
Can we disable the buttons and Position alternating?

### DISCUSSION THREAD

**Discussion by gajrajantino** [prabhuignoto/react-chrono#233] [Q&A] [answered]
- Can I disable the 4 buttons, I have a static timeline hence I don't need those buttons.
- Also, is it possible to have alternating position of cards? (i.e. one is up, next is down and one is left, one is right in horizontal and vertical modes respectively)

**Comment by prabhuignoto** [comment, marked as answer]
> Can I disable the 4 buttons, I have a static timeline hence I don't need those buttons.

use `hideControls` to disable the button controls.

> Also, is it possible to have alternating position of cards? (i.e. one is up, next is down and one is left, one is right in horizontal and vertical modes respectively)

Please use the `VERTICAL_ALTERNATING` mode to alternate cards in vertical mode.
There is no alternating available for Horizontal view at the moment. But defeinitely sounds like great addition as this is one of the most requested feature recently.

**Reply by gajrajantino** [reply]
Thanks!
Yea, the alternating horizontal view would be a great addition!
